### DISCUSSION TITLE
Content Security Policy (CSP) - Allow all

### DISCUSSION THREAD

**Discussion by alexander-schneider** [nuxt/nuxt#9749] [Questions (Nuxt 2)]
Hello Folks,

we are a very dynamic (user-generated website) and I want to allow everything in my CSP at first (and then make some fine-tuning). Now I have this code:

`  render: {
    csp: {
      hashAlgorithm: 'sha256',
      policies: {
         'default-src': [
            '*',
            "'unsafe-inline'",
            "'unsafe-eval'"
         ],
         'script-src': [
            '*',
            "'unsafe-inline'",
            "'unsafe-eval'"
         ],
         'connect-src': [
            '*',
            "'unsafe-inline'"
         ],
         'img-src': [
            '*',
            "'unsafe-inline'"
         ],
        'style-src': [
            '*',
            "'unsafe-inline'"
         ],
        'frame-src': [
            '*'
         ]
      },
      addMeta: true
    },`

But: No chance. There is still a lot of blocks. Does anybody have experience here?

Best regards
Thx

**Comment by JHerwig** [comment]
1. Please format your code for better reading (Use "```" instead of "`" for block code)
2. If you allow everything, why add CSP? 
3. Give `reportOnly` a try:
```
renderer: {
    csp: {
      reportOnly: true,
      // ...
      }
}
```

This should print you the detailed information in the console what would be blocked and why. Then you can adjust your settings.
Hint: Chrome is more detailed than Firefox.
